src/index.html

```diff
@@ -11,7 +11,134 @@
+        <label class="form_field">
+        Surname:
+          <input
+            type="text"
+            name="surname" 
+            autocomplete="off"
+          >
+        </label>
+        <label class="form_field">
+        Name:
+          <input
+            type="text" 
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form_field">
+        How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form_field">
+        Full date of birth:
+          <input type="date" name="birthday">
+        </label>
+        <label class="form_field">
+        I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            required 
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form_field">
+        Password:
+          <input
+            type="password" 
+            name="password" 
+            minlength="5"
+            maxlength="20"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <label class="form_field">
+        Do you love cats?
+        <input
+          type="radio"
+          name="cats"
+          value="Yes"
+        >
+        Yes
+          <input 
+            type="radio"
+            name="cats" 
+            value="No"
+          >
+        No 
+        </label>
+        <label class="form_field">
+        What is your favorite color?
+          <input type="color"  name="fav_color">
+        </label>
+        <label class="form_field">
+        What time do you go to bed?
+          <input type="time"  name="time">
+        </label>
+        <label class="form_field">
+        What is your favorite brands of cars?
+          <select  name="cars" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+            <option value="empty"></option>
+          </select>
+        </label>
+        <label class="form_field">
+        How do you rate our work?
+          <input 
+            type="range"
+            name="rate" 
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Additional info:</legend>
+        <label class="form_field">
+        Comments
+          <textarea 
+            name="comments"
+            rows="2" 
+            cols="20">
+          </textarea>
```

Comment:
use this to prevent empty space when focusing in input first time
```suggestion
            cols="20"></textarea>
```